scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryClient.scala

```diff
@@ -447,22 +450,22 @@ class BigQueryClient private (private val projectId: String,
   }
 
   /** Extract tables to be accessed by a query. */
-  def extractTables(sqlQuery: String): Set[TableReference] = {
+  def extractTables(sqlQuery: String): Option[Set[TableReference]] = {
```

Comment:
Instead of `Option[Set[]]`, is it cleaner to return empty set if `tryJob.get.getStatistics.getQuery.getReferencedTables == null`?